dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -107,6 +111,9 @@ module.exports = Promise.resolve()${this.compileFrontendModuleImports(frontendMo
     protected compileElectronMain(): string {
         return `// @ts-check
 
+const { resolve } = require('path');
+const { isMaster } = require('cluster');
```

Comment:
You may want to rebase as clustering has just been removed.

Reply:
Nice catch, I think I got confused during a rebase :)